src/arch/armv8/armv8-a/cpu.c

```diff
@@ -27,7 +27,15 @@ void cpu_arch_profile_init(cpuid_t cpuid, paddr_t load_addr)
     }
 }
 
-void cpu_arch_profile_idle()
+void cpu_arch_profile_standby()
+{
+    int32_t err = psci_standby();
+    if (err) {
+        ERROR("PSCI cpu%d standby failed with error %ld", cpu()->id, err);
+    }
+}
+
+void cpu_arch_profile_powerdown()
```

Comment:
In the recent changes, we implemented checks for active interrupts before calling the `powerdown` function, as detailed in this [commit](https://github.com/AFOliveira/bao-hypervisor-mem-throttling/commit/b908dd5e80dadf6ec6046c8d469625bd1428651a). The same logic applies to the `standby` function.

@josecm, do you think this approach is also appropriate in this context?

Reply:
I believe that is not needed for now. We only need to check if an interrupt is being handled if you go to idle/powerdown in the middle of an interrupt handler, which is not the case at the moment.